schemas/did-uuid.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) 2017-2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+
+const pattern = '^did:v1:uuid:([-_A-Za-z0-9.]+)$';
```

Comment:
Should this regex and the urn-uuid one strictly match a normal v4 UUID?

Reply:
Sure, you have a regex for that handy?  Also: https://github.com/veres-one/veres-one-validator/issues/11

Shall I assign that issue to you?

Reply:
The interwebs suggest something like the following for v4 UUIDs.  I only tested it briefly by hand.
```suggestion
const pattern = '^did:v1:uuid:([a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12})$';
```

Reply:
If found this, not sure if it's UUIDV4 specific, but seems like a credible source:
https://github.com/epoberezkin/ajv/blob/master/lib/compile/formats.js#L18